comit/src/network/protocols/orderbook/order.rs

```diff
@@ -78,6 +81,16 @@ fn meaningless_expiry_value() -> u32 {
     100
 }
 
+/// The position the maker takes in this order. A BTC/DAI buy order,
+/// also described as an order that buys the trading pair BTC/DAI,
+/// means that the order maker buys the base currency (in this case
```

Comment:
```suggestion
/// means that the maker buys the base currency (in this case
```

Reply:
Here I am specifically trying to link the maker of the order with the position i.e., the position is from the perspective of person who makes the order.

Reply:
Ok, now I get it but I find it quite confusing to read TBH :sweat_smile: 

It is sort of implied that they are making an order and not cake so I would still drop it :)

Reply:
lol, I'll take some cake, make cake, take cake - definitely take.

Reply:
Will try to better English this whole comment.

Reply:
Is this clearer?
```
/// The position of the maker for this order. A BTC/DAI buy order,
/// also described as an order that buys the trading pair BTC/DAI,
/// means that the maker buys the base currency (in this case BTC) in
/// return for DAI. A sell order means that the maker sells BTC and
/// receives DAI.
///
/// Please note: we do not set the base currency to 1 and use rate
/// (i.e., quote currency) and amount as is commonly done in Forex
/// trading. We use the amounts of each currency to determine the
/// rate.
```

@lucas, you are the resident English expert - can you read over this comment please and give your input?

Reply:
That one is clear to me :)